src/main/scala/dsptools/Chisel3TestersCompatibility.scala

```diff
@@ -0,0 +1,14 @@
+package chisel3.iotesters
```

Comment:
I thought Record was originally to address the fact that I wanted Vecs with non-uniform widths? addPin was a totally separate thing. 

I get that Chisel is short-handed as far as developers are concerned. Unfortunately, "gradware" methodology probably worked best for me. As a Chisel user, waiting weeks/months/infinite time for a feature I want is kind of unrealistic, since, frankly speaking, I'm not paid by the hour, and needing Chisel with w/e features to be up to some certain standard (or just stable enough -- which it currently isn't) just so I can actually get my PhD (which isn't in tools + methodology) is really painfully frustrating. Case in point being memory stuff, which blew up in my face during the last tapeout. 

From your standpoint, it makes sense to support larger bodies rather than one off projects, simply b/c there just aren't enough man hours to support everyone within due time. If I actually had spare time and no urgent deadlines, then yeah, I wouldn't mind trying to work with people to find a better solution than punching holes. But if I'm supposed to be using this stuff to tapeout in <3 months, then that's not really viable, and to meet such a deadline, unless I get something like ~day support turnaround, hacking is really the only thing I can do. That's pretty much why dsptools was/is a mess. Just dump stuff where we can and deal with it later. From an actual hardware design risk mitigation standpoint, it makes more sense for me to be working in my Chisel2 framework, but that's not what the powers that be want me to do...

 [I can go into another long rant about how tapeout + infrastructure/tools development should be totally separate things and the former should not happen until the latter is on much more solid ground (even if the former kind of *should* inform the latter about how things should be done -- at least from a user experience POV), but that's... again... out of my control...]